dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -513,53 +539,96 @@ CDefaultContextMenu::AddStaticContextMenusToMenu(
             continue;
         }
 
-        BOOL Extended = FALSE;
+        UINT cmdFlags = 0;
+        BOOL hide = FALSE;
         HKEY hkVerb;
         if (idResource > 0)
         {
-            if (LoadStringW(shell32_hInstance, idResource, wszVerb, _countof(wszVerb)))
-                mii.dwTypeData = wszVerb; /* use translated verb */
-            else
-                ERR("Failed to load string\n");
-
-            LONG res = RegOpenKeyW(info.hkClass, wszKey, &hkVerb);
-            if (res == ERROR_SUCCESS)
+            if (!(uFlags & CMF_OPTIMIZEFORINVOKE))
             {
-                res = RegQueryValueExW(hkVerb, L"Extended", NULL, NULL, NULL, NULL);
-                Extended = (res == ERROR_SUCCESS);
-
-                RegCloseKey(hkVerb);
+                if (LoadStringW(shell32_hInstance, idResource, wszVerb, _countof(wszVerb)))
+                    mii.dwTypeData = wszVerb; /* use translated verb */
+                else
+                    ERR("Failed to load string\n");
```

Comment:
What's the fallback value used in dwTypeData in that case?

Reply:
```
 /* By default use verb for menu item name */
mii.dwTypeData = (LPWSTR)info.Verb.GetString();
```